src/App.js

```diff
@@ -1,10 +1,194 @@
+import { useState } from 'react';
+import axios from 'axios';
+import Swal from 'sweetalert2';
+import withReactContent from 'sweetalert2-react-content'
+
+// components
+import UserOptions from './Components/UserOptions';
+
+// stylesheet
+import './styles/sass/App.scss'
+
+const MySwal = withReactContent(Swal)
 
 function App() {
+  const [boredResponse, setBoredResponse] = useState('');
+  const [showForm, setShowForm] = useState(true);
+  const [userTypeChoice, setUserTypeChoice] = useState('');
+  const [userPriceChoice, setUserPriceChoice] = useState('');
+  const [userNumberChoice, setUserNumberChoice] = useState('');
+  const [disableNumberMenu, setDisableNumberMenu] = useState(false);
+  const [showStyling, setShowStyling] = useState(false);
+
+  const handleUserTypeChoice = (e) => {
+    setUserTypeChoice(e.target.value)
+    // the BoredAPI only has "solo" activities for these, so the number/people menu is set to "solo" and greyed out (logic below, in return statement)
```

Comment:
I really like how you managed the pseudocode of your project, it's very easy to read and understand.